### PR TITLE
src: more verbose output

### PR DESCRIPTION
**Pull request by matejvasek**
# Changes

- :broom:  Log cluster events and cluster containers logs in GH Action. This will be useful for debugging of failed actions.

### CODE REVIEW

File: .github/workflows/test-integration.yaml [hunk 1]
```diff
@@ -43,6 +43,16 @@ jobs:
         run: ./hack/patch-hosts.sh
       - name: Integration Test
         run: make test-integration
+      - name: Dump Cluster Logs
+        if: always()
```
Comment (matejvasek) [inline .github/workflows/test-integration.yaml:47]: This also could have been `failure()` but I think it shoudln't hurt having `always()`.

**Review by lkingland — APPROVED** [review]

**Comment by matejvasek** [comment]
@zroubalik @lance we need to override the linter error here.

File: hack/binaries.sh [hunk 2]
```diff
@@ -72,4 +73,11 @@ install_helm() {
     helm version
 }
 
+install_stern() {
+  echo 'Installing stern...'
+  curl -sSL "https://github.com/stern/stern/releases/download/v1.25.0/stern_1.25.0_linux_amd64.tar.gz" | \
```
Comment (lance) [inline hack/binaries.sh:78]: How about adding a `local stern_version=1.25.0` in `main()`?

**Review by lance — COMMENTED** [review]
looks good - wdyt about making `stern_version` a varaible?

**Comment by matejvasek** [comment]
@lance PTAL

**Comment by matejvasek** [comment]
@lance the linter error has to be overridden.

**Pull request merged**